spec.md

```diff
@@ -380,13 +380,43 @@ Example error response (invalid `id` provided):
 
 ## Collections
 
+A collection is a namespace for Resources, and a unit of configuration, within
+a space.
+
+In other storage systems, a collection is also known as: Directory, Folder,
+RDBMS Table, Document Collection, Graph, WebAPI FileList, Bucket, Solid
+Container, EDV Vault, DWN Collection, and so on.
+
+The use of collections is optional -- when a Space is created, the default `/`
+collection is automatically created within that Space.
+
 ### Collection Data Model
 
-#### Collection JSON Representation (ActivityPub profile)
+Collection properties:
 
-### Collection Operations
+* `id`
+* `name` - human readable shortname, url slug
+* `type` - A sorted array of strings, MUST include the type `Collection`.
+* `contents`
+* (tbd) a link to the **system meta** auxiliary object
+* (tbd) one or more links to controller-modifiable **custom meta** auxiliary
```

Comment:
See my earlier comment where I don't think we should use these 'meta' terms until there is a PR that introduces the whole concept in a way that makes sense to me, is rooted in a use case, and can't be implemented using one of the other affordances we've implemented/discussed.

Reply:
K, removing them for now.